### DISCUSSION TITLE
HOWTO for VPS

### DISCUSSION THREAD

**Discussion by team-boo** [open-webui/open-webui#7093] [General]
i want to install on VPS with respective ssl certificate without Docker.

https://mydomain.com

There is no HOWTO anywhere.

Even without VPS i have errors like 

"2:33:58 PM [vite-plugin-svelte] /home/mydomain/public_html/open-webui/src/lib/components/common/Folder.svelte:13:12 Folder has unused export property 'id'. If it is for external reference only, please consider using `export const id`
2:33:59 PM [vite-plugin-svelte] /home/mydomain/public_html/open-webui/src/lib/components/chat/Settings/Chats.svelte:17:11 Chats has unused export property 'saveSettings'. If it is for external reference only, please consider using `export const saveSettings`
2:33:59 PM [vite-plugin-svelte] /home/mydomain/public_html/open-webui/src/lib/components/layout/Help/HelpMenu.svelte:10:11 HelpMenu has unused export property 'showDocsHandler'. If it is for external reference only, please consider using `export const showDocsHandler`
2:34:00 PM [vite-plugin-svelte] /home/mydomain/public_html/open-webui/src/lib/components/chat/Settings/General.svelte:9:11 General has unused export property 'getModels'. If it is for external reference only, please consider using `export const getModels`
2:34:00 PM [vite-plugin-svelte] /home/mydomain/public_html/open-webui/src/lib/components/layout/Navbar.svelte:29:11 Navbar has unused export property 'title'. If it is for external reference only, please consider using `export const title`
2:34:02 PM [vite-plugin-svelte] /home/mydomain/public_html/open-webui/src/lib/components/chat/Controls/Controls.svelte:10:11 Controls has unused export property 'models'. If it is for external reference only, please consider using `export const models`
2:34:03 PM [vite-plugin-svelte] /home/mydomain/public_html/open-webui/src/lib/components/layout/Navbar/Menu.svelte:24:11 Menu has unused export property 'shareEnabled'. If it is for external reference only, please consider using `export const shareEnabled`
2:34:03 PM [vite-plugin-svelte] /home/mydomain/public_html/open-webui/src/lib/components/layout/Navbar/Menu.svelte:26:11 Menu has unused export property 'downloadHandler'. If it is for external reference only, please consider using `export const downloadHandler`
2:34:03 PM [vite-plugin-svelte] /home/mydomain/public_html/open-webui/src/lib/components/chat/MessageInput/CallOverlay.svelte:664:5 A11y: <video> elements must have a <track kind="captions">
2:34:05 PM [vite-plugin-svelte] /home/mydomain/public_html/open-webui/src/lib/components/chat/Messages/ResponseMessage.svelte:49:11 ResponseMessage has unused export property 'rateMessage'. If it is for external reference only, please consider using `export const rateMessage`"

Thank you.

**Comment by comeback01** [comment]
I didn't understand the problem.

What exactly would you like to do?

**Comment by team-boo** [comment]
I want to install Open WebUI on vps https://mydomain.com

There are other VPS on the server.

VPS Managment is done from Virtualmin.
